### ISSUE TITLE
install fails for vst and dssi plugins: No such file or directory

### ISSUE THREAD

**Issue by yurivict** [linuxmao-org/fogpad#2]
For the VST plugin:
```
make install -C plugins/Fogpad
gmake[2]: Entering directory '/usr/ports/audio/fogpad-lv2/work/fogpad-port-1.0.0-1-g618cbaf/plugins/Fogpad'
install: /usr/ports/audio/fogpad-lv2/work/stage/usr/local/lib/vst/fogpad-vst.so: No such file or directory
```

For the DSSI plugin:
```
gmake install -C plugins/Fogpad
gmake[3]: Entering directory '/usr/ports/audio/fogpad-lv2/work/fogpad-port-1.0.0-1-g618cbaf/plugins/Fogpad'
install: /usr/ports/audio/fogpad-lv2/work/stage/usr/local/lib/dssi/fogpad-dssi.so: No such file or directory
gmake[3]: *** [Makefile:138: install] Error 71
gmake[3]: Leaving directory '/usr/ports/audio/fogpad-lv2/work/fogpad-port-1.0.0-1-g618cbaf/plugins/Fogpad'
gmake[2]: *** [Makefile:59: install] Error 2
gmake[2]: Leaving directory '/usr/ports/audio/fogpad-lv2/work/fogpad-port-1.0.0-1-g618cbaf'
```

**Comment by jpcima** [comment]
This problem seems to be an incompatibility with the tool `install` of FreeBSD.

I'm thinking of two potential solutions
1. patch the `Makefile` to invoke the GNU version of this command (aka `ginstall`)
2. rewrite it in a way which is compatible with both OS

**Comment by yurivict** [comment]
The directory it's looking for the files seems to be wrong.

**Comment by jpcima** [comment]
The error is about the target directory of the copy not existing.
On GNU install, when -D is passed, this creates the directory.
It seems that BSD install interprets this differently, hence this command fails.

**Comment by yurivict** [comment]
You can just split the directory creating into a separate ```mkdir``` command.

**Comment by jpcima** [comment]
It's resolved in c52326b

**Comment by yurivict** [comment]
Thanks!
